web-platform/website/src/components/navigation-bar/NavigationBar.tsx

```diff
@@ -70,11 +70,11 @@ function DropdownMenus(props: { user?: User }) {
 
   return (
     <>
-      <Col className="d-flex me-0 justify-content-end d-lg-none d-xl-none d-xxl-none">
+      <Col className="d-flex me-0 justify-content-end d-lg-none">
         <MobileMenu />
       </Col>
 
-      <Col className="d-flex me-0 justify-content-end d-none d-lg-flex d-xl-flex d-xxl-flex">
+      <Col className="d-flex me-0 justify-content-end d-none d-lg-flex">
```

Comment:
the first `d-flex` here is redundant since it is immediately overwritten by `d-none`

Reply:
Done